unity-client/Assets/Tutorial/Tests/TutorialControllerShould.cs

```diff
@@ -0,0 +1,181 @@
+using DCL.Tutorial;
+using NUnit.Framework;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace DCL.Tutorial_Tests
+{
+    public class TutorialControllerShould : TestsBase
+    {
+        private int currentStepIndex = 0;
+        private List<TutorialStep> currentSteps = new List<TutorialStep>();
+
+        protected override IEnumerator TearDown()
+        {
+            foreach (var step in currentSteps)
+            {
+                GameObject.Destroy(step);
+            }
+
+            tutorialController.SetTutorialDisabled();
+            tutorialController.stepsOnGenesisPlaza.Clear();
+            tutorialController.stepsFromDeepLink.Clear();
+            tutorialController.stepsOnGenesisPlazaAfterDeepLink.Clear();
+            currentSteps.Clear();
+            currentStepIndex = 0;
+
+            yield return base.TearDown();
+        }
+
+        [UnityTest]
+        public IEnumerator ExecuteTutorialStepsFromGenesisPlazaCorrectly()
+        {
+            ConfigureTutorialForGenesisPlaza();
+
+            yield return CoroutineStarter.Start(tutorialController.StartTutorialFromStep(0));
```

Comment:
Is `CoroutineStarter.Start` necessary? We can't just do `yield return tutorialController.StartTutorialFromStep(0)`?

Reply:
You are completely right... it was a little confussion :D Done! Thanks!